src/common/components/ReviewSection/ReviewSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { PenIcon } from "@/common/components/CustomIcon";
+import { ReviewItem } from "@/common/components/ReviewItem";
+import { reviewSectionTheme } from "@/common/components/ReviewSection/ReviewSection.theme";
```

Comment:
chore(non-blocking): missed a relative import here

```suggestion
import { reviewSectionTheme } from "./ReviewSection.theme";
```

Reply:
thanks! fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/7cdf46559f65df476ed690af89d3dedc6d9716c8